src/maintainer/maintainer_faq.rst

```diff
@@ -86,5 +86,24 @@ FAQ
       - ``conda install boa -c conda-forge``
       - ``conda mambabuild myrecipe``
     For more details visit `this <https://boa-build.readthedocs.io/en/latest/mambabuild.html>`_ page. 
+    
+    .. _mfaq_conda_verify:
+
+:ref:`(Q) <mfaq_conda_verify>` **I am seeing** ``Importing conda-verify failed.`` **error message during build. What do I do?**
+
+   ``Importing conda-verify failed. Please be sure to test your packages. conda install conda-verify to make this message go away.``
+You are seeing this error message because by default, conda-build uses conda-verify to ensure that your recipe and package meet some minimum sanity checks. 
+This message can be safely ignored. 
```

Comment:
Thankyou Bastian. :)
I have one doubt though.
I don't understand what we mean by 'conda-forge doesn't use it'. 
So, can I write, "This message can be safely ignored because conda-forge doesn't use conda-verify" ?

Reply:
Yeah, something along those lines.

Conda-verify is an optional dependency of conda-build so whoever installs conda-build can either install conda-verify or not (Which is the case in the conda-forge infrastructure): https://github.com/conda/conda-build/blob/3b99b2222a067e113a2282926871cd1e5406ee2b/conda.recipe/meta.yaml#L30

During execution, conda-build will try to import conda-verify and run it, otherwise, if conda-build cannot find (Meaning import) conda-verify, it will just skip the verification.
https://github.com/conda/conda-build/blob/3b99b2222a067e113a2282926871cd1e5406ee2b/conda_build/build.py#L1683-L1688

Since the scripts used to set up the conda-forge CI infrastructure and local test environment do not install conda-verify, the latter is the case.

So long story short: The conda-forge infrastructure does not install conda-verify as it is not needed and therefore that part is skipped. :)

Reply:
Wowwwww!
Gottt ittt! <3

Reply:
Thanks so explaining so well.

Reply:
Thanks for the comment! I couldn't have done it better

Reply:
Happy to help :)